src/Pages/Home.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import mockup from "../Images/mockup.png";
+import sunIcon from "../Images/icon-sun.png";
+import "../App.css";
+import Button from "../Components/Button";
+import CourseTopicCard from "../Components/CourseTopicCard";
+import "../Components/styles.css";
+
+export default function Home() {
+    return (
+        <div className="App">
+            <header className="App-header">
+                <CourseTopicCard
+                    icon={sunIcon}
+                    intro="PRODUCT THINKING"
+                    header="Designing the right thing"
+                    description="Learn how to find the right problem space, do real user reserach, and ideate a variety of solutions using industry processes."
+                    img={mockup}
```

Comment:
it seems like the font family broke for the course topic card? perhaps a css change

Reply:
Good catch! I'll look into that.